packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -33,6 +35,8 @@ export class ApiBuilder {
 
   private verifyAuthMiddleware?: Middleware;
 
+  private translatorConfigured: boolean = false;
```

Comment:
This is very subjective so entirely optional, but to me initialised just speaks more to the idea that it happens once on first use
```suggestion
  private translatorInitialised: boolean = false;
```

Reply:
Usually I would agree with this particular language. I used "configured" here to match the actual functions we're calling against the i18n library. We call `i18n.configure` once, but `i18n.init` against each req.